map/src/main/res/layout/fragment_map.xml

```diff
@@ -128,6 +128,26 @@
         </LinearLayout>
     </FrameLayout>
 
+    <FrameLayout
+        android:orientation="horizontal"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
```

Comment:
Can't this `ImageView` just exist directly within the `ConstraintLayout`, without a frame parent?